Makefile

```diff
@@ -188,7 +188,7 @@ bundle: manifests kustomize ## Generate bundle manifests and metadata, then vali
 
 .PHONY: bundle-build
 bundle-build: ## Build the bundle image.
-	docker build -f bundle-custom.Dockerfile -t $(BUNDLE_IMG) .
+	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
```

Comment:
This was the sole user of `bundle-custom.Dockerfile`. It should be removed in this commit as well IMHO.

Reply:
Prow CI will use bundl-custom.Dockerfile.  It has to stay.

Reply:
> Prow CI will use bundl-custom.Dockerfile. It has to stay.

I realized/remembered about that when you provided your status during the daily standup. This should be documented somewhere in the repo.